orcidhistories.md

```diff
@@ -0,0 +1,53 @@
+# ORCID History Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Single ORCID History entry
+**GET /api/eperson/orcidhistories/<:id>**
```

Comment:
Similar to orcidqueue, this endpoint should have the following notes added:
* We should add a note to say this endpoint is only available when orcid.sychronization-enabled=true
* We should also note who has permissions to access this endpoint. It's only the person who is the owner of the Profile Item who can access it.
* We should note the relationship between this endpoint and orcidqueues.  So, add a note that after the queue is processed it ends up in this list